ProjectLighthouse.Servers.GameServer/Controllers/CommentController.cs

```diff
@@ -97,8 +97,7 @@ from blockedProfile in this.database.BlockedProfiles
             .ApplyPagination(pageData)
             .ToListAsync()).ToSerializableList(c => GameComment.CreateFromEntity(c, token.UserId));
 
-        if (type == CommentType.Level && slotType == "developer" && user.IsModerator && pageData.PageStart == 1)
-        {
+        if (type == CommentType.Level && slotType == "developer" && user.IsModerator && pageData.PageStart == 1) {
```

Comment:
```suggestion
        if (type == CommentType.Level && slotType == "developer" && user.IsModerator && pageData.PageStart == 1)
        {
```

I assume your formatter did this